ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/AgingImagesChart.tsx

```diff
@@ -125,7 +125,7 @@ function AgingImagesChart({ searchFilter, timeRanges, timeRangeCounts }: AgingIm
                 width={widgetContainerResizeEntry?.contentRect.width} // Victory defaults to 450
                 padding={{
                     top: 25,
-                    left: 55,
+                    left: 65,
```

Comment:
Follow up after MVP - these values should be dynamic in some way to account for changing data. Right now the SVG padding is a best guess at what will work for most systems.